src/TuringHelpers.jl

```diff
@@ -1,4 +1,5 @@
-using .Turing, .MCMCChains
+#using .Turing, .MCMCChains
```

Comment:
I think it is safe to remove this commented code.

Reply:
Done.